multicluster/cmd/gateways.go

```diff
@@ -41,7 +41,14 @@ func newGatewaysCommand() *cobra.Command {
 				return err
 			}
 
-			client, err := public.NewExternalClient(cmd.Context(), controlPlaneNamespace, k8sAPI)
+			ctx := cmd.Context()
+
+			vizNs, err := k8sAPI.GetNamespaceWithExtensionLabel(ctx, "linkerd-viz")
+			if err != nil {
+				return err
```

Comment:
For now should this produce a more helpful error explaining this relies on the Viz extension being installed?